fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanApplicationWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -2478,7 +2478,15 @@ private void saveCashFlowProjection(Integer projectionRate, BigDecimal amount, L
             LoanCashFlowData cashFlow) {
         LoanCashFlowProjection projection = new LoanCashFlowProjection(installment.getInstallmentNumber(), cashFlow.getId(), projectionRate,
                 amount);
-        this.loanCashFlowProjectionRepository.saveAndFlush(projection);
+        try {
```

Comment:
@Deepika1095  Can you add this exception in the logic not intercept the error log. This will be miss-leading and not safe approach

Reply:
@hero78 I made changes as discussed, please check.